src/Microsoft.TestPlatform.PlatformAbstractions/uap10.0/System/PlatformFile.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.PlatformAbstractions
+{
+    using System.IO;
+
+    /// <summary>
+    /// File Abstraction
+    /// </summary>
+    public static class PlatformFile
+    {
+        /// <summary>
+        /// Checks if give file exists on disk
+        /// </summary>
+        /// <param name="filePath">input filePath</param>
+        /// <returns>True if file Exists</returns>
+        public static bool Exists(string filePath)
+        {
+            // Appxrecipe gets renamed vs.appxrecipe, which test platform is unaware of, so if queried for appxrecipe, return true
+            if (filePath.EndsWith(".appxrecipe", System.StringComparison.OrdinalIgnoreCase))
```

Comment:
This is business logic getting in platform layer :) Would the consumer of platform APIs expect such filtering?

Reply:
This is the most unfortunate hack I'm introducing since we have to go back to doing UWP discovery in fullCLR process, just like I predicted it would come back to haunt me.
I hated doing this, but business requirement trumps.